addons/sourcemod/scripting/szf/infected.sp

```diff
@@ -859,9 +855,12 @@ public void Infected_OnJockeyThink(int iClient, int &iButtons)
 
 public void Infected_OnJockeyTouch(int iClient, int iToucher)
 {
-	//Already pouncing someone and must be in air to pounce
-	if (0 < g_iJockeyTarget[iClient] <= MaxClients || GetEntityFlags(iClient) & FL_ONGROUND || !IsValidLivingSurvivor(iToucher))
+	//Must not already be latched onto someone and be pouncing
+	if (0 < g_iJockeyTarget[iClient] <= MaxClients || !g_bJockeyIsUsingPounce[iClient] || GetEntityFlags(iClient) & FL_ONGROUND || !IsValidLivingSurvivor(iToucher))
+	{
+		g_bJockeyIsUsingPounce[iClient] = false;
```

Comment:
There may be an edge case where `g_bJockeyIsUsingPounce` is set to false by touching non-client entity.
Should only be set to false when jockey is actually on ground.

Reply:
This was an issue that was already present in Hunter's pounce, both now make use of think callbacks to work around it. Also seems to fix touch detection when landing on top of survivors.